src/components/RegionFlaggedContent.js

```diff
@@ -0,0 +1,36 @@
+import * as clipboard from 'clipboard-polyfill/text'
+import { useState } from 'react'
+import { h, span } from 'react-hyperscript-helpers'
+import { Link } from 'src/components/common'
+import { icon } from 'src/components/icons'
+import { regionInfo } from 'src/components/region-common'
+import { TooltipCell } from 'src/components/table'
+import { withErrorReporting } from 'src/libs/error'
+import * as Utils from 'src/libs/utils'
+
+
+// Put a google-region appropriate flag next to some content (or a region description by default, if content is undefined).
+const RegionFlaggedContent = ({ location, locationType, content, clipboardText, children, ...props }) => {
```

Comment:
I think it possibly made more sense before I extracted out the region functions (because there was one case where I couldn't re-use this component anyway). Separate code paths seems logical to me.